.github/workflows/nightly.yaml

```diff
@@ -28,20 +28,14 @@ jobs:
         with:
           java-version: 11
 
-      - name: Cache gradle dependencies
+      - name: Test
```

Comment:
While we're here how about `Test on ${{matrix.java}}`?

Reply:
We already have in job name. Isn't it enough?